### PR TITLE
[translation] Update Dutch(nl-NL) 

### PR DESCRIPTION
**Pull request by Jensderond**
Translated some english words.

### Types of changes
Textual change

### CODE REVIEW

File: src/i18n/locales/nl.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
   "settings.service.form.enableService": "Activeer service",
   "settings.service.form.enableNotification": "Activeer meldingen",
   "settings.service.form.team": "Team",
-  "settings.service.form.customUrl": "Custom server",
+  "settings.service.form.customUrl": "Aangepaste server",
```
Comment (adlk) [inline src/i18n/locales/nl.json]: My dutch skills are non existent but I'm not sure if 'Aangepaste' would be the proper translation as this label should reflect the meaning of:
- I'm having my own server
- I'm using an on-premise server

Comment (mroeling) [reply src/i18n/locales/nl.json]: Technically "custom" is not a word that exists in Dutch. So it should have been translated to something else.
However, as @adlk states correctly, "custom" means more than just "aangepast", and in this context I would prefer something like "Eigen/lokale server" as alternative string. More like literally the meaning you state, Stefan. 
"Aangepast" would be translated back to "Changed", and that is not correct in this context.

File: src/i18n/locales/nl.json [hunk 2]
```diff
@@ -101,7 +101,7 @@
   "settings.service.form.editServiceHeadline": "Bewerk {name}",
   "settings.service.form.tabHosted": "Gehost",
   "settings.service.form.tabOnPremise": "Zelf-gehoste ??",
-  "settings.service.form.customUrlValidationError": "Kon custom server {name} niet valideren.",
+  "settings.service.form.customUrlValidationError": "Kon aangepaste server {name} niet valideren.",
```
Comment (mroeling) [inline src/i18n/locales/nl.json]: Please read comment below about this translation.

**Comment by Jensderond** [comment]
Thanks for the feedback I've updated the pr! 👍

**Pull request merged**